compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -347,52 +351,61 @@ void addPadPackBasedPassPipeline(OpPassManager &pm,
     AMDAIEBufferizeToAllocationOptions bufferizeOptions2;
     bufferizeOptions2.memorySpace = 2;
     bufferizeOptions2.bufferizeOperand = BufferizeOperand::Input;
-    modulePassManager.addNestedPass<func::FuncOp>(
+    modulePassManager.addPass(
         createAMDAIEBufferizeToAllocationPass(bufferizeOptions2));
   }
-
+  /*
   // Lower to UKernels
   {
     AMDAIELowerToUKernelsOptions options;
     options.passPipeline = AIEPassPipeline::PadPackPipeline;
     options.pathToUkernels = clPathToUkernels;
-    modulePassManager.addNestedPass<func::FuncOp>(
+    modulePassManager.addPass(
         createAMDAIELowerToUKernelsPass(options));
   }
-
+  */
   // Vectorization passes
   appendVectorizationToPipeline(modulePassManager);
 
   // Comprehensive bufferization
   addAMDAIEBufferizePasses(modulePassManager);
-  modulePassManager.addPass(createLowerUKernelOpsToCallsPass());
+  //modulePassManager.addPass(createLowerUKernelOpsToCallsPass());
 }
 
-void buildAMDAIETransformPassPipeline(OpPassManager &pm) {
-  addCommonTargetExecutablePreprocessingPasses(pm);
+void buildAMDAIETransformPassPipeline(OpPassManager &variantPassManager) {
+
+  OpPassManager &modulePassManager = variantPassManager.nest<ModuleOp>();
+  modulePassManager.addPass(createLowerExecutableUsingTransformDialectPass());
+  {
+    FunctionLikeNest funcPassManager(modulePassManager);
+    addCommonTargetExecutablePreprocessingPasses(funcPassManager);
+  }
   {
     AMDAIELoweringStrategyOptions options;
     options.usePassPipeline = clUsePipeline;
     options.numCores = clNumCores;
-    pm.addPass(createAMDAIELoweringStrategyPass(options));
+    modulePassManager.addPass(createAMDAIELoweringStrategyPass(options));
   }
   {
+    FunctionLikeNest funcPassManager(modulePassManager);
```

Comment:
Before this (i.e. after LoweringStrategyPass) the `createLowerExecutableUsingTransformDialectPass` should run.